python/lsst/ts/idl/enums/ATPtg.py

```diff
@@ -125,3 +126,9 @@ class TargetTypes(enum.IntEnum):
     AZEL = 2
     PLANET = 3
     EPHEM = 4
+
+
+class WrapStrategy(enum.IntEnum):
```

Comment:
A doc string explaining these would be useful. I think I can guess what 1 and 3 are, but I don't know what OPTIMIZE  optimizes (slew time?).